doc/api/errors.md

```diff
@@ -2085,6 +2085,11 @@ unknown file extension.
 
 The requested functionality is not supported in worker threads.
 
+<a id="ERR_WORKER_UNSUPPORTED_PROTOCOL"></a>
+### `ERR_WORKER_UNSUPPORTED_PROTOCOL`
+
+The requested protocol is not supported in worker threads.
+
```

Comment:
I’d much rather re-use `ERR_INVALID_URL_SCHEME` here, i.e. the same error that the fs operations throw